qa/tasks/scripts/install_ceph_salt.sh

```diff
@@ -0,0 +1,21 @@
+set -ex
+
+REPO=$1
+BRANCH=$2
+if [ -n "$REPO" ]
+then
+	if [ ! -n "$BRANCH" ]
+	then
+		BRANCH="master"
+	fi
```

Comment:
This can be replaced with single command:
```
BRANCH=${BRANCH:-"master"}
```
And in fact, it is better to move it outside of the _if_ when you declare this variable:
```
BRANCH=${2:-"master"}
```